docs/support.md

```diff
@@ -58,7 +56,7 @@ through [email](mailto:support@semgrep.com).
 
 Users of Semgrep OSS can log bugs and feature requests in the
 [semgrep](https://github.com/semgrep/semgrep/issues) repository. They can also
-ask questions regarding usage, rollouts, and deployments in the [community Slack
+ask questions regarding usage, roll outs, and deployments in the [community Slack
```

Comment:
I think it is "rollsouts"?

Reply:
Vale did me dirty. Will add rollouts to the allowlist.